homeassistant/components/waqi/sensor.py

```diff
@@ -89,7 +89,7 @@
 class WaqiSensor(Entity):
     """Implementation of a WAQI sensor."""
 
-    def __init__(self, client, station):
+    def __init__(self, client, station, token):
```

Comment:
How many stations can you configure with 1 token? You can just copy paste it twice in your config and update 2 stations, no ?

Reply:
As many stations as a user would like.  Locations and stations are accepted as a list.  I will make sure to attach `self.uid` to account for additional entities.